common/lib/capa/capa/inputtypes.py

```diff
@@ -818,8 +818,15 @@ def setup(self):
         self.setup_code_response_rendering()
 
     def _extra_context(self):
-        """Defined queue_len, add it """
-        return {'queue_len': self.queue_len, }
+        """Define queue_len, arial_label and code mirror exit message context variables"""
```

Comment:
Always format docstrings using the multi-line convention, even if there’s only one line of content.

http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html?highlight=docstring#docstrings